### PR TITLE
W-11670348-addXrefs-duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/dataweave-cookbook-pluck.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
 
-The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
+In addition to using the DataWeave functions such as xref:dw-core-functions-entriesof.adoc[entriesOf], xref:dw-core-functions-keysof.adoc(keysOf), or xref:dw-core-functions-valuesof.adoc[valuesOf] to work with key-value pairs, you can also use dw-core-functions-pluck.adoc[pluck]. The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
```
Comment (mlischetti) [inline modules/ROOT/pages/dataweave-cookbook-pluck.adoc:7]: Just a small changes, replace (keysOf) by [keysOf]

**Review by mlischetti — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/dataweave-cookbook-pluck.adoc (hunk 1 above)
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-cookbook-pluck.adoc:7]:
```suggestion
In addition to using the DataWeave functions such as xref:dw-core-functions-entriesof.adoc[entriesOf], xref:dw-core-functions-keysof.adoc[keysOf], or xref:dw-core-functions-valuesof.adoc[valuesOf] to work with key-value pairs, you can also use dw-core-functions-pluck.adoc[pluck]. The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
```

**Review by mlischetti — APPROVED** [review]

File: modules/ROOT/pages/dataweave-cookbook-pluck.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: studio, anypoint, transform, transformer, format, dataweave, data weave, output structure, mime type, mime
 
-The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
+In addition to using the DataWeave functions such as xref:dw-core-functions-entriesof.adoc[entriesOf], xref:dw-core-functions-keysof.adoc[keysOf], or xref:dw-core-functions-valuesof.adoc[valuesOf] to work with key-value pairs, you can also use dw-core-functions-pluck.adoc[pluck]. The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
```
Comment (dukesphere) [inline modules/ROOT/pages/dataweave-cookbook-pluck.adoc:7]:
```suggestion
In addition to using the DataWeave functions such as xref:dw-core-functions-entriesof.adoc[entriesOf], xref:dw-core-functions-keysof.adoc[keysOf], or xref:dw-core-functions-valuesof.adoc[valuesOf] to work with key-value pairs, you can also use xref:dw-core-functions-pluck.adoc[pluck]. The following Mule app example shows how to use the DataWeave `pluck` function to extract key-value pairs from a JSON payload and store them into Mule event variables with variable names as the keys and their corresponding values.
```

File: modules/ROOT/pages/dataweave-cookbook-pluck.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/dataweave-cookbook-pluck.adoc]: done

**Review by mlischetti — APPROVED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**